src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -51,180 +51,170 @@ const defaultProps = {
-                                onPress={Report.navigateToConciergeChat}
-                                icon={Expensicons.ChatBubble}
-                                style={[styles.mt4]}
-                                iconStyles={[styles.buttonCTAIcon]}
-                                shouldShowRightIcon
-                                large
-                                success
-                            />
-                            <MenuItem
-                                title={this.props.translate('workspace.bankAccount.noLetsStartOver')}
-                                icon={Expensicons.RotateLeft}
-                                onPress={BankAccounts.requestResetFreePlanBankAccount}
-                                shouldShowRightIcon
-                                wrapperStyle={[styles.cardMenuItem, styles.mv3]}
-                            />
-                        </Section>
-                        {this.props.reimbursementAccount.shouldShowResetModal && <WorkspaceResetBankAccountModal reimbursementAccount={this.props.reimbursementAccount} />}
-                        {!requiresTwoFactorAuth && <Enable2FAPrompt />}
-                    </ScrollView>
-                )}
-            </ScreenWrapper>
-        );
-    }
+                    )}
+                </Form>
+            )}
+            {isVerifying && (
+                <ScrollView style={[styles.flex1]}>
+                    <Section
+                        title={translate('workspace.bankAccount.letsFinishInChat')}
+                        icon={Illustrations.ConciergeBubble}
+                    >
+                        <Text>{translate('validationStep.letsChatText')}</Text>
+                        <Button
+                            text={translate('validationStep.letsChatCTA')}
+                            onPress={Report.navigateToConciergeChat}
+                            icon={Expensicons.ChatBubble}
+                            style={[styles.mt4]}
+                            iconStyles={[styles.buttonCTAIcon]}
+                            shouldShowRightIcon
+                            large
+                            success
+                        />
+                        <MenuItem
+                            title={translate('workspace.bankAccount.noLetsStartOver')}
+                            icon={Expensicons.RotateLeft}
+                            onPress={BankAccounts.requestResetFreePlanBankAccount}
+                            shouldShowRightIcon
+                            wrapperStyle={[styles.cardMenuItem, styles.mv3]}
+                        />
+                    </Section>
+                    {reimbursementAccount.shouldShowResetModal && <WorkspaceResetBankAccountModal reimbursementAccount={reimbursementAccount} />}
+                    {!requiresTwoFactorAuth && <Enable2FAPrompt />}
+                </ScrollView>
+            )}
+        </ScreenWrapper>
+    );
 }
 
 ValidationStep.propTypes = propTypes;
```

Comment:
@BartoszGrajdek We need to add displayName for functional components

https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#function-component-style

Reply:
Other than that code looks fine

Reply:
Gentle ping @BartoszGrajdek

Reply:
Hi, sorry for the delay - I took a sick leave for a couple of days. It's updated now :)